airbyte-integrations/connectors/source-datadog/source_datadog/schemas/monitors.json

```diff
@@ -0,0 +1,365 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "additionalProperties": true,
+  "properties": {
+    "created": {
+      "type": ["string", "null"]
+    },
+    "creator": {
+      "additionalProperties": true,
+      "properties": {
+        "email": {
+          "type": ["string", "null"]
+        },
+        "handle": {
+          "type": ["string", "null"]
+        },
+        "name": {
+          "type": ["string", "null"]
+        }
+      },
+      "type": ["object", "null"]
+    },
+    "deleted": {
+      "type": "null"
```

Comment:
Please specify the data type

Reply:
Yea this was a weird one. Documentation says this will always be `null` but does list type of date-time so I'll adjust accordingly.

Reply:
fixed